tikv-client/src/main/java/com/pingcap/tikv/TiDBJDBCClient.java

```diff
@@ -110,6 +110,44 @@ public boolean splitTableRegion(
     }
   }
 
+  public boolean splitIndexRegion(
+      String dbName, String tblName, String idxName, List<List<String>> splitPoints)
+      throws SQLException {
+    // SPLIT TABLE t1 INDEX idx4 ("a", "2000-01-01 00:00:01"), ("b", "2019-04-17 14:26:19"), ("c",
+    // "");
+    StringBuilder sb = new StringBuilder();
+    sb.append("split table ")
+        .append(dbName)
```

Comment:
If you wish to compose the query via string, do not forget to wrap **`** around your table and database names.

Reply:
I will fix this.

Reply:
DONE